config/initializers/transcryptor.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
```

Comment:
@nattfodd Would it make more sense if this file just says
```
require 'db/transcryptor/migrations.rb'
```
or if Transcryptor is actually a Rails engine that when loaded, can automatically require the `'db/transcryptor/*.rb'` files?

Reply:
You mean on the per-file basis? So we would have:

```
-- db
---- transcryptor
------- user.rb
------- billing.rb
------- order.rb
```

Instead of:
```
-- config
---- initializers
------ transcryptor.rb
```
I believe that makes sense and could be done

Reply:
Hi @nattfodd , yes, that's great, thanks!